test/views/components/LateSubmissionPenaltySummaryCardSpec.scala

```diff
@@ -707,14 +707,13 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
       }
 
       "have the appeal status for REJECTED" in {
-        docWithAppealedPointRejected.select("dt").text().contains("Point due to expire") shouldBe true
```

Comment:
Should we have a check for this? I guess the penalty is a charge, right?

Reply:
What do you think to this?

Reply:
I am not sure what you are referring too/ what the issue is?

Reply:
This line was removed as part of your PR, I think it is a useful check we should have

Reply:
As it is a financial penalty it should not have an expiry date, I have updated the case data to be more clear on the penalty type